motoman_driver/MotoPlus/MpRosAllControllers.vcxproj

```diff
@@ -113,7 +113,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='YRC1000|Win32'">
     <NMakeBuildCommandLine>mpbuilder.exe -c $(Configuration) -p "$(ProjectDir)\" -n "MotoRosYRC1_" -o build</NMakeBuildCommandLine>
-    <NMakeOutput>MotoRosYRC1_.out</NMakeOutput>
+    <NMakeOutput>$(MP_VS_Install)DevTools\OnlineDownload.exe</NMakeOutput>

```

Comment:
Is this correct? Seems like a strange name for the output binary?

Reply:
Yes, this is correct. That field doesn't actually control the file name of the output binary (this isn't a true makefile project). This trick allows Visual Studio to launch the remote-upload utility when you build and attempt to "run" the application. So, after you build, you can easily push the updated binary into the robot controller without booting into maintenance mode. (This only works on YRC.)